proof-systems/src/darlin/t_dlog_acc_marlin/mod.rs

```diff
@@ -432,51 +426,27 @@ where
         pc_vk_g1: &<PC<G1, FS> as PolynomialCommitment<G1>>::VerifierKey,
         pc_vk_g2: &<PC<G2, FS> as PolynomialCommitment<G2>>::VerifierKey,
         public_input: &[G1::ScalarField],
-        prev_inner_sumcheck_acc: &DualSumcheckItem<G2, G1>,
-        prev_dlog_acc: &DualDLogItem<G2, G1>,
+        prev_acc: &DualTDLogItem<G2, G1, PC<G2, FS>, PC<G1, FS>>,
         proof: &Proof<G1, G2, FS>,
     ) -> Result<
         (
-            DualSumcheckItem<G1, G2>,
+            DualTDLogItem<G1, G2, PC<G1, FS>, PC<G2, FS>>,
             (
-                DensePolynomial<G1::ScalarField>,
                 DensePolynomial<G2::ScalarField>,
```

Comment:
Instead of returning this anonymous DensePolynomial, can we return instead DualTDlogItem::ExpandedItem ?

Reply:
No more necessary, after having simplified the interfaces of the prover and verifier (no more need to pass around the polynomials behind the accumulators).